db/schema.rb

```diff
@@ -12,12 +12,15 @@
 
 ActiveRecord::Schema.define(version: 20190228195434) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
   create_table "companies", force: :cascade do |t|
     t.string   "name"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.datetime "deleted_at"
-    t.index ["deleted_at"], name: "index_companies_on_deleted_at"
+    t.index ["deleted_at"], name: "index_companies_on_deleted_at", using: :btree
```

Comment:
remove using: btree

Reply:
I think this comes as a result of dumping the database so will not cause any problem.